src/storage/posix/storage.c

```diff
@@ -341,7 +263,8 @@ storagePosixMove(THIS_VOID,  StorageRead *source, StorageWrite *destination, Sto
             // Determine which file/path is missing
             if (errno == ENOENT)
             {
-                if (!storageInterfaceExistsP(this, sourceFile))
+                // Check if the source is missing. Rename does not follow links so there is no need to set followLink.
+                if (!storageInterfaceInfoP(this, sourceFile, storageInfoLevelExists).exists)
```

Comment:
But `storageInterfaceInfoP(this, sourceFile, storageInfoLevelExists).exists` does not check for whether this is a file or a directory and in the old code, `storageInterfaceExistsP(this, sourceFile)` would return false if it were a directory, so this is a change in behavior -was that intentional?

Reply:
This fixes an existing bug where a move on path with a missing destination would error on path missing rather than creating the destination.

Error should be `unable to move missing source`.